### PR TITLE
Refactor preview-name and collapsibles

### PR DESCRIPTION
**Pull request by aliciaizmar**
Hello! @irenegf83  I've made the following things:
**preview-name.js:**
Arrow function
code indentation

**collapsibles.js**
Remove comments and blank spaces
create some constant to avoid large content
code indentation

### CODE REVIEW

File: _src/assets/js/collapsibles.js [hunk 1]
```diff
@@ -1,26 +1,24 @@
-/* eslint-disable strict */
-
-const legends = document.querySelectorAll('.legend');
-const collapsibles = document.querySelectorAll('.collapsibles');
+const legends = document.querySelectorAll(".legend");
+const collapsibles = document.querySelectorAll(".collapsibles");
```
Comment (irenegf83) [inline _src/assets/js/collapsibles.js]: delete this line

File: _src/assets/js/collapsibles.js [hunk 2]
```diff
@@ -1,26 +1,24 @@
...
 
 for (const legend of legends) {
-  legend.addEventListener('click', handlebuttonclick);
+  legend.addEventListener("click", handlebuttonclick);
 }
 
 function handlebuttonclick(event) {
```
Comment (irene-gomez) [inline _src/assets/js/collapsibles.js]: Convert this function to arrow function

**Review by irene-gomez — CHANGES_REQUESTED** [review]
I just put you to turn the `handlebuttonclick` function into an arrow. And I suppose the `addEventListener` will have to go to the end of that arrow function.

File: _src/assets/js/collapsibles.js (hunk 2 above)
Comment (aliciaizmar) [reply _src/assets/js/collapsibles.js]: Hi, changes are made!

**Review by irene-gomez — APPROVED** [review]

**Review by irene-gomez — APPROVED** [review]
Merge!! ;)

**Pull request merged**